packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -13,28 +13,21 @@ import {
   FileInfoQuery,
   Hbar,
   HbarUnit,
-  PrecheckStatusError,
   Query,
   Status,
   Transaction,
-  TransactionId,
   TransactionRecord,
   TransactionRecordQuery,
   TransactionResponse,
 } from '@hashgraph/sdk';
-import { EventEmitter } from 'events';
+import EventEmitter from 'events';
```

Comment:
nit: use `{ EventEmitter }` for consistency

Reply:
Fixed.